modules/apps/analytics/analytics-client-js/test/utils/validators.ts

```diff
@@ -28,7 +34,8 @@ describe('isValidEvent()', () => {
 	});
 
 	it('returns if an event is valid', () => {
-		const event = {
+		const event: Event = {
+			applicationId: '123',
```

Comment:
Hi Didi @interaminense ,
while I was converting this test, it kept accusing me of missing this parameter, but I'm not sure if it's correct to add it like this. Can you take a look at it? :relaxed: